Sources/Extensions/SwiftStdlib/DictionaryExtensions.swift

```diff
@@ -30,8 +30,8 @@ public extension Dictionary {
     ///		dict.keys.contains("key2") -> false
     ///
     /// - Parameter keys: keys to be removed
-    public mutating func removeAll(keys: [Key]) {
-        keys.forEach({ removeValue(forKey: $0)})
+    public mutating func removeAll<S: Sequence>(keys: S) where S.Element == Key {
+        keys.forEach({ removeValue(forKey: $0) })
```

Comment:
My old comment seems to have disappeared, but I think it's better to remove the parenthesis for `forEach`. I.e. `forEach { removeValue(forKey: $0) }`. It's what autocomplete does and I think SwiftLint complains about it too.

Reply:
Swiftlint doesn't complain, but no problem changing it

Reply:
Not all SwiftLint rules are applied by default. You're welcome to add it: `unneeded_parentheses_in_closure_argument` ;)
https://github.com/realm/SwiftLint/blob/master/Source/SwiftLintFramework/Rules/UnneededParenthesesInClosureArgumentRule.swift